src/routes/JobBoard/jobs.json

```
@@ -0,0 +1,12 @@
+[
+    {
+        "title": "Software Engineer",
+        "company": "Amazon Web Services",
```

```suggestion
        "company": "Example Job",
```

Actually, before we merge this, let's put in a fake company. We don't want to ship to production with something confusing